integration/testnet/network.go

```diff
@@ -467,29 +485,39 @@ func PrepareFlowNetwork(t *testing.T, networkConf NetworkConfig) *FlowNetwork {
 
 	// create a temporary directory to store all bootstrapping files, these
 	// will be shared between all nodes
```

Comment:
```suggestion
	// create a temporary directory to store all bootstrapping files 
```
this directory isn't shared any more given 
https://github.com/onflow/flow-go/blob/4587d40755e58c89c56ad66f2ecf79372fc4fcb6/integration/testnet/network.go#L632-L637